lib/models/user.dart

```diff
@@ -0,0 +1,19 @@
+class TheUser {
```

Comment:
z zasady, plik darta powinien się nazywać tak samo jak klasa która w nim się znajduje. można stosować kebab-case lub underscore_case w zależności od preferencji (ja preferuję to drugie), byle było jednolicie.

Reply:
okej, done